Modules/itertoolsmodule.c

```diff
@@ -1834,12 +1617,26 @@ chain_new(PyTypeObject *type, PyObject *args, PyObject *kwds)
     return chain_new_internal(type, source);
 }
 
+/*[clinic input]
+@classmethod
+itertools.chain.from_iterable
+
+    iterable: object
+    /
+
+Create a chain object.
+
+Alternate chain() contructor taking a single iterable argument
```

Comment:
Spelling: _constructor_

(Also, _alternative_ is more correct than _alternate_ outside the US.)